app/templates/me/index.hbs

```diff
@@ -31,8 +31,10 @@
 
     <p>
         If you want to use package commands from the command line, you will need to
-        login with <code>cargo login</code> using one of the tokens listed below. <br/>
-        When working in shared environments, supplying the token on the command line could expose it to other users, so it is more secure to supply a token when prompted. 
+        login with <code>cargo login (token)</code> using one of the tokens listed below. <br/>
+        When working in shared environments, supplying the token on the command line could 
+        expose it to prying eyes via command line history. To avoid this, enter <code>cargo login</code>
```

Comment:
Ok fair enough, yeah noise is an issue too, think we'll leave `ps -a` out since it'd take an extra sentence to explain it. Sorry to negate your latest change :)